src/Makefile

```diff
@@ -65,20 +65,22 @@ ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
 ifeq ($(USE_POLLY_ACC),1)
-LLVMLINK += -lPollyPPCG
+LLVMLINK += -lPollyPPCG -lGPURuntime
 FLAGS += -DUSE_POLLY_ACC
-ifeq ($(CUDALIB_INCLUDE_DIR),)
-$(error CUDALIB_INCLUDE_DIR not set: If compiling with USE_POLLY_ACC, user must provide the path to CUDA header files)
-endif
-FLAGS += -I$(CUDALIB_INCLUDE_DIR)
-FLAGS += -I$(shell $(LLVM_CONFIG_HOST) --src-root)/tools/polly/tools
+FLAGS += -I$(shell $(LLVM_CONFIG_HOST) --src-root)/tools/polly/tools # Required to find GPURuntime/GPUJIT.h
 endif
 endif
 else
 SRCS += anticodegen
 LLVM_LIBS := support
 endif
 
+$(build_shlibdir)/libGPURuntime.$(SHLIB_EXT):
+ifeq ($(USE_POLLY_ACC), 1)
+# Place libGPURuntime such that it's accessible under libjulia's RPATH
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.$(SHLIB_EXT) $(build_shlibdir)
+endif
```

Comment:
Personally, I'm not sure we include support for `USE_SYSTEM_*` configurations that aren't from distributions.  This seems like a custom setup that another user may not necessarily replicate properly.  I'm more inclined to support either a build from within Julia's source tree, or only binaries sourced from package managers.